javarosa/src/main/java/org/javarosa/xpath/expr/XPathPositionFunc.java

```diff
@@ -54,4 +54,12 @@ private static Double position(TreeReference refAt) {
         return new Double(refAt.getMultLast());
     }
 
+    @Override
+    public String getDocumentation() {
+        return getDocHeader()
+                + "Behavior: Returns the current index of the given reference. \n"
+                + "Return: the greatest extant multiplicity of the reference. \n"
+                + "Arguments: The reference to be checked, or none for current position "
+                + "Example: position(/data/repeat) returns '2' if two iterations have been created";
```

Comment:
@wpride We should remove documentation for the position($arg) form of this method. It is unsupported and has ambiguous confusing behavior.